inst/@sym/solve.m

```diff
@@ -101,11 +120,10 @@
             '    else:'
             '        symbols.append(arg)'
             '#'
-            'if len(symbols) > 0:'
-            '    d = sp.solve(eqs, symbols, dict=True)'
-            'else:'
-            '    d = sp.solve(eqs, dict=True)'
+            'd = sp.solve(eqs, *symbols, dict=True)'
             '#'
+            'if not isinstance(d, (list, tuple)):'
+            '    return d,'
```

Comment:
what does this change do?

You asked why not comfortable with the previous refactor; this is one reason, new things happening that I don't understand.  Does this fix something?  If so, there is no test.